### PR TITLE
Use Telephone for all TTY numbers

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/11418

This PR adds the `<Telephone />` component for all TTY numbers for consistency.

## Testing done
Updated unit tests

## Screenshots
N/A

## Acceptance criteria
- [x] Use `<Telephone />` for all TTY number links.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by Mottie — APPROVED** [review]

**Review by jbalboni — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Review by erikphansen — COMMENTED** [review]
I'm sure you're on this already but there are a lot of test failures that can be resolved by setting `pattern={PATTERNS['911']}`, but perhaps the `911` pattern should be renamed since it now applies to the 711 number, too.

**Review by bkjohnson — COMMENTED** [review]
I didn't see any updated unit tests like you mentioned in "Testing Done", but overall I think this is a welcome change. I'll let all the app codeowners approve it first before the frontend team gives it the :+1:

**Comment by kelsonic** [comment]
@bkjohnson I didn't add unit tests yet, but if you look at Jenkins failing tests I think I will inevitably have to modify a few 😂

**Comment by kelsonic** [comment]
@department-of-veterans-affairs/frontend-review-group ready for review! 🎉

**Comment by kelsonic** [comment]
The `PATTERNS['911']` syntax is being proposed to be changed to `PATTERNS[3_DIGIT]` in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/11781) 🙂

File: src/applications/personalization/profile-2/components/connected-apps/AdditionalInfoSections.jsx
Comment (Clark-W-Griswold) [inline src/applications/personalization/profile-2/components/connected-apps/AdditionalInfoSections.jsx:119]: Was this content change intended?

File: src/applications/personalization/profile360/components/MilitaryInformation.jsx
Comment (Clark-W-Griswold) [inline src/applications/personalization/profile360/components/MilitaryInformation.jsx:131]: Did we intend to replace this or was the addition of a new phone number intended?

**Review by Clark-W-Griswold — COMMENTED** [review]
Left some comments for review.

File: src/applications/personalization/profile-2/components/connected-apps/AdditionalInfoSections.jsx
Comment (kelsonic) [reply src/applications/personalization/profile-2/components/connected-apps/AdditionalInfoSections.jsx:119]: Nope lol thanks!

**Review by ncksllvn — APPROVED** [review]

File: src/applications/personalization/profile360/components/MilitaryInformation.jsx
Comment (kelsonic) [reply src/applications/personalization/profile360/components/MilitaryInformation.jsx]: Updated! Thanks! 🙂

**Review by Clark-W-Griswold — APPROVED** [review]
👍

File: src/platform/static-data/CallHRC.jsx
Comment (zurbergram) [inline src/platform/static-data/CallHRC.jsx:12]: Typo in the word "hearing"

File: src/platform/site-wide/va-footer/components/CrisisPanel.jsx
Comment (ncksllvn) [inline src/platform/site-wide/va-footer/components/CrisisPanel.jsx:77]: @kelsonic I think we have a link within a link here, producing this error

image

**Comment by kelsonic** [comment]
Thanks for the catch! I'll send up a PR for these 2 fixes now 🙂

**Pull request merged**
